src/transformers/onnx/features.py

```diff
@@ -58,27 +61,110 @@ class FeaturesManager:
     }
 
     # Set of model topologies we support associated to the features supported by each topology and the factory
-    _SUPPORTED_MODEL_KIND = {
-        "albert": supported_features_mapping("default", onnx_config_cls=AlbertOnnxConfig),
-        "bart": supported_features_mapping("default", onnx_config_cls=BartOnnxConfig),
-        "mbart": supported_features_mapping("default", onnx_config_cls=MBartOnnxConfig),
-        "bert": supported_features_mapping("default", onnx_config_cls=BertOnnxConfig),
+    _SUPPORTED_MODEL_TYPE = {
+        "albert": supported_features_mapping(
+            "default",
+            "masked-lm",
+            "sequence-classification",
+            # "multiple-choice",
```

Comment:
Should we delete this bit of dead code? (And similarly for the other occurrences of `multiple-choice`?)

Reply:
I wonder because the plan is to add support for this too one day.
I do not know how much work that represents.

Reply:
OK let's keep it then :)